docs/docsite/rst/contributing/creating_collections.rst

```diff
@@ -0,0 +1,138 @@
+.. _creating_content_collections:
+
+********************
+Creating Collections
+********************
+
+.. contents:: Topics
+
+This topic describes how to build a collection and distribute it through Galaxy.
+
+.. _creating_collections:
+
+Collections
+===========
+
+Collections are a distribution format for Ansible content. They can be used to
+package and distribute roles, modules and plugin types.
+
+.. note::
+    This is a Tech-Preview feature. Future Galaxy releases may introduce breaking changes.
+
+
+Collection Metadata
+===================
+
+Collections require a ``galaxy.yml`` at the root level of the collection. This file contains all of the metadata that Galaxy
+and mazer need in order to package and import a collection.
+
+.. note::
+    mazer will only accept ``.yml`` extensions for galaxy.yml.
+
+::
+
+    collection/
+    ├── README.md
+    ├── galaxy.yml
+    ├── plugins/
+    │   ├── modules/
+    │   │   └── module1.py
+    │   ├── inventory/
+    │   └── .../
+    └── roles/
+        ├── role1/
+        └── role2
+
+
+``galaxy.yml`` requires the following fields in order to be accepted by galaxy:
+
+.. code-block:: yaml
+
+    namespace: "namespace_name"
+    name: "collection_name"
+    version: "1.0.12"
+    readme: "README.md"
+    authors:
+        - "Author1"
+        - "Author2 (http://author2.example.com)"
+        - "Author3 <author3@example.com>"
+    dependencies:
+        - "collection.name1"
+        - "collection.name2"
```

Comment:
dependencies is a dictionary, maybe change namespace from "collection" to something else, maybe give multiple example specs:
```
    dependencies:
        "other_namespace.collection1": ">=1.0.0"   
        "other_namespace.collection2": ">=2.0.0,<3.0.0"
        "anderson55.my_collection": "*"    # note: "*" selects the highest version available
```